tests/output/LoggingTest.py

```diff
@@ -3,6 +3,16 @@
 import unittest
 
 from coalib.output.Logging import configure_json_logging
+from coalib.output.printers.LogPrinter import LogPrinterMixin
+
+
+class TestLogger(LogPrinterMixin):
+
+    def __init__(self):
+        pass
+
+    def log_message(self, log_message, **kwargs):
+        super(TestLogger, self).log_message(log_message)
```

Comment:
(if it's for coverage, please add a note in the commit body that to keep up coverage, you had to write a test for the deprecated components that can't be removed yet)